command_test.go

```diff
@@ -708,9 +708,6 @@ func TestEmptyInputs(t *testing.T) {
 }
 
 func TestOverwrittenFlag(t *testing.T) {
```

Comment:
Good call.  I think I'm going with the "shadow" terminology, since that is what was used in the issue description.

I think it also more accurately describes the situation because the parent flag still exists (it was not literally overwritten), it was just superseded by the child flag.